frontend/src/components/chat/Message.vue

```diff
@@ -38,6 +49,12 @@ export default defineComponent({
 		};
```

Comment:
I cant comment on line 45 but it says:

45: // shouldbe removed in favour of sockets + init once 

is this something we can do now?